packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -116,6 +119,7 @@ export const SelectArrayInput = (props: SelectArrayInputProps) => {
     } = props;
 
     const inputLabel = useRef(null);
+    const translateLabel = useTranslateLabel();
```

Comment:
`translateLabel` is no longer used so you should remove it

Reply:
@slax57 removed the ununsed line